docs/0.5/howto/removing_a_splinter_circuit.md

```diff
@@ -0,0 +1,366 @@
+[Configuring Splinter Nodes](
+{% link docs/0.5/tutorials/configuring_splinter_nodes.md %}). In this guide,
+you'll be taking on the role of two different admins for two nodes (on the same
+system), so that you can create and test removing a circuit. For the first part
+of the guide, you will act as the first "alpha" node administrator to
+propose disbanding a circuit. Then, you will act as the second "beta" node
+administrator to accept the disband proposal. For the second part of this
+guide, you'll act as the first "alpha" node administrator to abandon a
+circuit. Next, you'll act as the second "beta" node administrator to view the
+"ABANDONED CIRCUIT" message from the "alpha" node. Finally, you will purge the
+inactive circuit.
+
+## Prerequisites
+
+* A splinter network with at least two nodes running and at least one circuit
+  created. The guide will refer to the two nodes as "alpha" and "beta"
+  respectively. The guide will refer to the already created circuit as
+  "01234-ABCDE". For more information on configuring a Splinter network, see
+  [Configuring Splinter Nodes]({%link
+  docs/0.5/tutorials/configuring_splinter_nodes.md %})
+
+* The splinter command line tool installed with
+  [experimental features]({% link docs/0.5/howto/building_splinter.md %})
+  enabled
+
+## Procedure
+A circuit may be removed from networking by either disbanding the circuit or
+abandoning the circuit. Although this guide follows both processes, disbanding
+and abandoning are alternative options. A circuit may be purged regardless of
+the operation used to deactivate the circuit. The first part of this procedure
+will explain how to disband a circuit. The next part will explain how to
+abandon a circuit. Finally, the inactive circuit will be purged.
+
+### Alternate 1: Disbanding a Circuit
+
+The first approach this tutorial will explain is disbanding a circuit to remove
+it’s networking capabilities. This is an alternative to abandoning a circuit,
+which does not involve all circuit members. A circuit may be abandoned OR
+disbanded. The following instructions begin with the alpha node administrator
+requesting to disband a circuit.
+
+1. Request to disband a circuit by running the `splinter-circuit-disband`
+   command from the alpha node.
+
+    ``` console
+    $ splinter circuit disband 01234-ABCDE --url http://splinterd-alpha:8080 \
+        --key <path_to_alpha_private_key>
+    ```
+
+   This command will create a circuit proposal with the circuit ID specified
+   in the command. This proposal will have a `ProposalType` of `Disband` to
+   differentiate it from proposals used to create a circuit.
+
+1. Connect to the beta Splinter node to finish disbanding the circuit.
+
+    a. View the disband proposal using the `splinter-circuit-proposals`
+      command.
+
+    ```console
+    $ docker exec -it splinterd-beta bash
+    root@beta:/# splinter circuit proposals \
+      --url http://splinterd-beta:8080 \
```

Comment:
Avoid using --url and --key in examples; instead, add a section before the commands which clarifies that url and key must be specified if they are not the default (and env variables are not used to initialize them).

Reply:
I added a bullet point for each of these options to explain that they are necessary if no default is present, otherwise pointing out the other options available to set these in the section before all of the command examples